src/composables/topHeader.ts

```diff
@@ -0,0 +1,35 @@
+import { computed, ref } from '@vue/composition-api';
+import type { Store } from 'vuex';
+import type { ITopHeader } from '../types';
+import { createPollingBasedOnMountedComponents } from './composablesHelpers';
+import { useSdk } from './sdk';
+
+interface UseTopHeaderOptions {
+  /**
+   * TODO: Temporary solution to avoid dependency circle
```

Comment:
Recently I had a dependency cycle in types, and I think it is ok to import such dependency using `import type` then linter won't blame it. This would be a cycle dependency only on the stage of type checking. Would it be ok or would it complicate unit testing or so?

Reply:
It is possible to import not the whole store but only it's type using `import type`?